File: Frontend/src/data/dummyData.js
export const DUMMY_USER = {
  id: "usr_101",
  name: "Alex Morgan",
  email: "alex.morgan@globetrotter.io",
  avatar: "https://images.unsplash.com/photo-1534528741775-53994a69daeb?auto=format&fit=crop&w=300&q=80",
  bio: "Passionate globetrotter, photographer, and culture enthusiast.",
  savedDestinations: ["Paris", "Zurich", "Tokyo", "Bali"],
  currency: "INR (₹)",
  language: "English"
};

export const DUMMY_TRIPS = [
  {
    id: "trip_1",
    name: "Europe Adventure",
    startDate: "2026-09-12",
    endDate: "2026-09-20",
    description: "An unforgettable multi-city tour through Paris, Zurich, and Rome featuring scenic trains, fine dining, and iconic historical landmarks.",
    coverImage: "https://images.unsplash.com/photo-1502602898657-3e91760cbb34?auto=format&fit=crop&w=1200&q=80",
    citiesCount: 3,
    daysCount: 8,
    estimatedBudget: 85500,
    status: "Upcoming",
    stops: [
      {
        id: "stop_1",
        cityName: "Paris",
        country: "France",
        flag: "🇫🇷",
        startDate: "2026-09-12",
        endDate: "2026-09-14",
        activities: [
          { id: "act_1", time: "09:00", name: "Airport Arrival & Hotel Check-in", category: "Transport", cost: 0, icon: "bi-airplane" },
          { id: "act_2", time: "11:00", name: "Eiffel Tower Sightseeing", category: "Sightseeing", cost: 2200, icon: "bi-building" },
          { id: "act_3", time: "14:00", name: "Lunch at Le Petit Bistro", category: "Food", cost: 1350, icon: "bi-cup-hot" },
          { id: "act_4", time: "18:00", name: "Seine River Cruise at Sunset", category: "Sightseeing", cost: 2700, icon: "bi-water" }
        ]
      },
      {
        id: "stop_2",
        cityName: "Zurich",
        country: "Switzerland",
        flag: "🇨🇭",
        startDate: "2026-09-15",
        endDate: "2026-09-17",
        activities: [
          { id: "act_5", time: "10:00", name: "Lake Zurich Boat Tour", category: "Sightseeing", cost: 3500, icon: "bi-water" },
          { id: "act_6", time: "14:00", name: "Old Town Walk & Swiss Chocolate Tasting", category: "Food", cost: 2100, icon: "bi-bag-check" }
        ]
      },
      {
        id: "stop_3",
        cityName: "Rome",
        country: "Italy",
        flag: "🇮🇹",
        startDate: "2026-09-18",
        endDate: "2026-09-20",
        activities: [
          { id: "act_7", time: "09:30", name: "Colosseum Guided Tour", category: "Culture", cost: 2800, icon: "bi-bank" },
          { id: "act_8", time: "15:00", name: "Vatican Museums & Sistine Chapel", category: "Culture", cost: 3100, icon: "bi-palette" }
        ]
      }
    ],
    budgetBreakdown: {
      transport: 25000,
      accommodation: 30000,
      activities: 12500,
      meals: 18000,
      total: 85500
    }
  },
  {
    id: "trip_2",
    name: "Tropical Bali Getaway",
    startDate: "2026-11-05",
    endDate: "2026-11-12",
    description: "Relaxing beach resort, sacred water temples, Ubud monkey forest, and waterfall hiking.",
    coverImage: "https://images.unsplash.com/photo-1537996194471-e657df975ab4?auto=format&fit=crop&w=1200&q=80",
    citiesCount: 2,
    daysCount: 7,
    estimatedBudget: 62000,
    status: "Upcoming",
    stops: [
      {
        id: "stop_4",
        cityName: "Ubud",
        country: "Indonesia",
        flag: "🇮🇩",
        startDate: "2026-11-05",
        endDate: "2026-11-08",
        activities: [
          { id: "act_9", time: "08:00", name: "Sacred Monkey Forest Sanctuary", category: "Nature", cost: 800, icon: "bi-tree" },
          { id: "act_10", time: "11:00", name: "Tegallalang Rice Terraces", category: "Sightseeing", cost: 500, icon: "bi-sun" }
        ]
      }
    ],
    budgetBreakdown: {
      transport: 18000,
      accommodation: 24000,
      activities: 10000,
      meals: 10000,
      total: 62000
    }
  },
  {
    id: "trip_3",
    name: "Tokyo & Kyoto Explorer",
    startDate: "2026-04-10",
    endDate: "2026-04-20",
    description: "Cherry blossom season experience visiting Shinjuku, Shibuya, Fushimi Inari, and Mt. Fuji.",
    coverImage: "https://images.unsplash.com/photo-1503899036084-c55cdd92da26?auto=format&fit=crop&w=1200&q=80",
    citiesCount: 2,
    daysCount: 10,
    estimatedBudget: 110000,
    status: "Completed",
    stops: [],
    budgetBreakdown: {
      transport: 35000,
      accommodation: 45000,
      activities: 15000,
      meals: 15000,
      total: 110000
    }
  }
];

export const DUMMY_CITIES = [
  {
    id: "paris",
    name: "Paris",
    country: "France",
    flag: "🇫🇷",
    region: "Europe",
    costIndex: "$$$",
    popularity: 5,
    image: "https://images.unsplash.com/photo-1502602898657-3e91760cbb34?auto=format&fit=crop&w=800&q=80",
    description: "City of lights, world-class gastronomy, haute couture, and art landmarks."
  },
  {
    id: "tokyo",
    name: "Tokyo",
    country: "Japan",
    flag: "🇯🇵",
    region: "Asia",
    costIndex: "$$$",
    popularity: 5,
    image: "https://images.unsplash.com/photo-1503899036084-c55cdd92da26?auto=format&fit=crop&w=800&q=80",
    description: "Futuristic skyscrapers juxtaposed with tranquil historic temples and incredible street food."
  },
  {
    id: "dubai",
    name: "Dubai",
    country: "United Arab Emirates",
    flag: "🇦🇪",
    region: "Middle East",
    costIndex: "$$$$",
    popularity: 4,
    image: "https://images.unsplash.com/photo-1512453979798-5ea266f8880c?auto=format&fit=crop&w=800&q=80",
    description: "Ultra-modern luxury, desert safaris, soaring towers, and artificial islands."
  },
  {
    id: "bali",
    name: "Bali",
    country: "Indonesia",
    flag: "🇮🇩",
    region: "Asia",
    costIndex: "$$",
    popularity: 5,
    image: "https://images.unsplash.com/photo-1537996194471-e657df975ab4?auto=format&fit=crop&w=800&q=80",
    description: "Tropical paradise known for volcanic mountains, iconic rice paddies, beaches, and coral reefs."
  },
  {
    id: "zurich",
    name: "Zurich",
    country: "Switzerland",
    flag: "🇨🇭",
    region: "Europe",
    costIndex: "$$$$",
    popularity: 4,
    image: "https://images.unsplash.com/photo-1515488042361-ee00e0ddd4e4?auto=format&fit=crop&w=800&q=80",
    description: "Picturesque lakeside metropolis framed by the snow-capped Swiss Alps."
  },
  {
    id: "rome",
    name: "Rome",
    country: "Italy",
    flag: "🇮🇹",
    region: "Europe",
    costIndex: "$$$",
    popularity: 5,
    image: "https://images.unsplash.com/photo-1552832230-c0197dd311b5?auto=format&fit=crop&w=800&q=80",
    description: "Eternal City brimming with nearly 3,000 years of globally influential art and architecture."
  }
];

export const DUMMY_ACTIVITIES = [
  {
    id: "act_101",
    cityId: "paris",
    name: "Eiffel Tower Sightseeing",
    category: "Sightseeing",
    duration: "2 hours",
    cost: 2200,
    rating: 4.9,
    image: "https://images.unsplash.com/photo-1543349689-9a4d426bee8e?auto=format&fit=crop&w=600&q=80",
    description: "Ascend the world's most famous tower for breathtaking panoramic views across Paris."
  },
  {
    id: "act_102",
    cityId: "paris",
    name: "Louvre Museum Guided Tour",
    category: "Culture",
    duration: "3.5 hours",
    cost: 1800,
    rating: 4.8,
    image: "https://images.unsplash.com/photo-1499856871958-5b9627545d1a?auto=format&fit=crop&w=600&q=80",
    description: "Explore masterpieces including the Mona Lisa and Venus de Milo in the world's largest art museum."
  },
  {
    id: "act_103",
    cityId: "paris",
    name: "Montmartre Food & Wine Tasting",
    category: "Food",
    duration: "3 hours",
    cost: 3500,
    rating: 4.9,
    image: "https://images.unsplash.com/photo-1510812431401-41d2bd2722f3?auto=format&fit=crop&w=600&q=80",
    description: "Sample artisanal French cheeses, freshly baked croissants, and organic wines in historic Montmartre."
  },
  {
    id: "act_104",
    cityId: "zurich",
    name: "Lindt Home of Chocolate Experience",
    category: "Food",
    duration: "2 hours",
    cost: 1500,
    rating: 4.7,
    image: "https://images.unsplash.com/photo-1511381939415-e44015466834?auto=format&fit=crop&w=600&q=80",
    description: "Interactive chocolate museum featuring a giant chocolate fountain and Unlimited tasting room."
  }
];

export const DUMMY_STATS = {
  totalTrips: 12,
  countriesVisited: 8,
  citiesExplored: 15,
  totalSavedBudget: "₹2,57,500"
};

File: Flutter Frontend/lib/data/dummy_data.dart
import '../models/activity.dart';
import '../models/city.dart';
import '../models/expense.dart';
import '../models/itinerary_item.dart';
import '../models/trip.dart';
import '../models/trip_stop.dart';
import '../models/user.dart';

/// Realistic mock dataset for GlobeTrotter Frontend Phase 1
class DummyData {
  DummyData._();

  // Current Logged In User
  static const User dummyUser = User(
    id: 'usr-001',
    name: 'Alex Morgan',
    email: 'alex.morgan@wanderlust.io',
    avatarUrl:
        'https://images.unsplash.com/photo-1534528741775-53994a69daeb?w=400&auto=format&fit=crop&q=80',
    bio: 'Photographer & cultural wanderer. 24 countries explored, infinity to go ✈️📸',
    tripsCount: 8,
    countriesCount: 14,
    citiesCount: 28,
    travelScore: 920,
  );

  // Destinations / Cities
  static const List<City> dummyCities = [
    City(
      id: 'city-paris',
      name: 'Paris',
      country: 'France',
      imageUrl:
          'https://images.unsplash.com/photo-1502602898657-3e91760cbb34?w=800&auto=format&fit=crop&q=80',
      rating: 4.9,
      description: 'The City of Light, romance, iconic architecture, art, and world-class culinary masterpieces.',
      tag: 'Romantic',
      avgCostPerDay: 220.0,
      latitude: 48.8566,
      longitude: 2.3522,
      isTrending: true,
    ),
    City(
      id: 'city-zurich',
      name: 'Zurich',
      country: 'Switzerland',
      imageUrl:
          'https://images.unsplash.com/photo-1515488764276-beab7607c1e6?w=800&auto=format&fit=crop&q=80',
      rating: 4.8,
      description: 'Pristine alpine lakes, medieval altstadt, luxury shopping, and snow-capped mountain backdrops.',
      tag: 'Scenic',
      avgCostPerDay: 310.0,
      latitude: 47.3769,
      longitude: 8.5417,
      isTrending: false,
    ),
    City(
      id: 'city-rome',
      name: 'Rome',
      country: 'Italy',
      imageUrl:
          'https://images.unsplash.com/photo-1552832230-c0197dd311b5?w=800&auto=format&fit=crop&q=80',
      rating: 4.9,
      description: 'The Eternal City with thousands of years of ancient history, Roman ruins, and gelato on every corner.',
      tag: 'Historical',
      avgCostPerDay: 180.0,
      latitude: 41.9028,
      longitude: 12.4964,
      isTrending: true,
    ),
    City(
      id: 'city-tokyo',
      name: 'Tokyo',
      country: 'Japan',
      imageUrl:
          'https://images.unsplash.com/photo-1503899036084-c55cdd92da26?w=800&auto=format&fit=crop&q=80',
      rating: 4.95,
      description: 'Hyper-futuristic neon metropolis blended harmoniously with centuries-old Shinto shrines.',
      tag: 'Metropolis',
      avgCostPerDay: 210.0,
      latitude: 35.6762,
      longitude: 139.6503,
      isTrending: true,
    ),
    City(
      id: 'city-bali',
      name: 'Bali',
      country: 'Indonesia',
      imageUrl:
          'https://images.unsplash.com/photo-1537996194471-e657df975ab4?w=800&auto=format&fit=crop&q=80',
      rating: 4.85,
      description: 'Tropical paradise featuring lush rice terraces, sacred sea temples, coral reefs, and vibrant beach clubs.',
      tag: 'Tropical',
      avgCostPerDay: 95.0,
      latitude: -8.4095,
      longitude: 115.1889,
      isTrending: true,
    ),
    City(
      id: 'city-dubai',
      name: 'Dubai',
      country: 'UAE',
      imageUrl:
          'https://images.unsplash.com/photo-1512453979798-5ea266f8880c?w=800&auto=format&fit=crop&q=80',
      rating: 4.8,
      description: 'Gleaming desert oasis boasting record-breaking skyscrapers, luxury mega malls, and dune safaris.',
      tag: 'Luxury',
      avgCostPerDay: 260.0,
      latitude: 25.2048,
      longitude: 55.2708,
      isTrending: false,
    ),
    City(
      id: 'city-santorini',
      name: 'Santorini',
      country: 'Greece',
      imageUrl:
          'https://images.unsplash.com/photo-1570077188670-e3a8d69ac5ff?w=800&auto=format&fit=crop&q=80',
      rating: 4.9,
      description: 'Whitewashed cliffside villages, caldera views, cobalt-blue domes, and world-renowned Aegean sunsets.',
      tag: 'Island',
      avgCostPerDay: 240.0,
      latitude: 36.3932,
      longitude: 25.4615,
      isTrending: true,
    ),
    City(
      id: 'city-newyork',
      name: 'New York',
      country: 'USA',
      imageUrl:
          'https://images.unsplash.com/photo-1496442226666-8d4d0e62e6e9?w=800&auto=format&fit=crop&q=80',
      rating: 4.85,
      description: 'The city that never sleeps, with Broadway lights, iconic Central Park, and global cuisine.',
      tag: 'Urban',
      avgCostPerDay: 280.0,
      latitude: 40.7128,
      longitude: -74.0060,
      isTrending: false,
    ),
  ];

  // Activities
  static const List<Activity> dummyActivities = [
    Activity(
      id: 'act-eiffel',
      cityId: 'city-paris',
      name: 'Eiffel Tower Sunset Summit & Champagne',
      category: 'Sightseeing',
      price: 85.0,
      duration: '3 hours',
      rating: 4.9,
      reviewsCount: 420,
      imageUrl:
          'https://images.unsplash.com/photo-1511739001486-6bfe10ce785f?w=600&auto=format&fit=crop&q=80',
      description: 'Skip-the-line access to the top floor of the Iron Lady with panoramic twilight city views.',
      location: 'Champ de Mars, 5 Av. Anatole France, Paris',
      isFeatured: true,
    ),
    Activity(
      id: 'act-louvre',
      cityId: 'city-paris',
      name: 'Louvre Masterpieces Guided Art Tour',
      category: 'Culture',
      price: 65.0,
      duration: '2.5 hours',
      rating: 4.8,
      reviewsCount: 310,
      imageUrl:
          'https://images.unsplash.com/photo-1565099824688-e93eb20fe622?w=600&auto=format&fit=crop&q=80',
      description: 'Explore the Mona Lisa, Venus de Milo, and iconic wings with a certified art historian.',
      location: 'Rue de Rivoli, Paris',
      isFeatured: true,
    ),
    Activity(
      id: 'act-seine',
      cityId: 'city-paris',
      name: 'Bateaux Parisiens Gourmet Dinner Cruise',
      category: 'Dining',
      price: 110.0,
      duration: '2 hours',
      rating: 4.85,
      reviewsCount: 195,
      imageUrl:
          'https://images.unsplash.com/photo-1543349689-9a4d426bee8e?w=600&auto=format&fit=crop&q=80',
      description: 'Glide along the Seine tasting 4-course French culinary creations under illuminated bridges.',
      location: 'Port de la Bourdonnais, Paris',
      isFeatured: false,
    ),
    Activity(
      id: 'act-colosseum',
      cityId: 'city-rome',
      name: 'Colosseum Underground & Gladiator Arena',
      category: 'History',
      price: 90.0,
      duration: '3.5 hours',
      rating: 4.95,
      reviewsCount: 680,
      imageUrl:
          'https://images.unsplash.com/photo-1552832230-c0197dd311b5?w=600&auto=format&fit=crop&q=80',
      description: 'Access the underground dungeons where gladiators prepped for combat and the arena floor.',
      location: 'Piazza del Colosseo, Rome',
      isFeatured: true,
    ),
    Activity(
      id: 'act-vatican',
      cityId: 'city-rome',
      name: 'Vatican Museums & Sistine Chapel Early Access',
      category: 'Art & History',
      price: 75.0,
      duration: '3 hours',
      rating: 4.9,
      reviewsCount: 520,
      imageUrl:
          'https://images.unsplash.com/photo-1531572753322-ad063cecc140?w=600&auto=format&fit=crop&q=80',
      description: 'Beat the crowds into Michelangelo\'s ceiling fresco and St. Peter\'s Basilica.',
      location: 'Viale Vaticano, Rome',
      isFeatured: true,
    ),
    Activity(
      id: 'act-shibuya',
      cityId: 'city-tokyo',
      name: 'Shibuya Crossing & Izakaya Food Alley Hop',
      category: 'Food & Nightlife',
      price: 70.0,
      duration: '3 hours',
      rating: 4.9,
      reviewsCount: 280,
      imageUrl:
          'https://images.unsplash.com/photo-1542051841857-5f90071e7989?w=600&auto=format&fit=crop&q=80',
      description: 'Navigate the legendary crossing and sample authentic yakitori, ramen, and craft sake.',
      location: 'Shibuya, Tokyo',
      isFeatured: true,
    ),
    Activity(
      id: 'act-ubud-swing',
      cityId: 'city-bali',
      name: 'Ubud Jungle Swing & Tegalalang Rice Terraces',
      category: 'Adventure',
      price: 45.0,
      duration: '4 hours',
      rating: 4.8,
      reviewsCount: 340,
      imageUrl:
          'https://images.unsplash.com/photo-1537996194471-e657df975ab4?w=600&auto=format&fit=crop&q=80',
      description: 'Soar above lush tropical canopies with panoramic valley views and artisan coffee tasting.',
      location: 'Tegalalang, Ubud, Bali',
      isFeatured: true,
    ),
  ];

  // Itinerary items for Grand Euro Tour Day 1-3
  static const List<ItineraryItem> dummyItinerary = [
    ItineraryItem(
      id: 'itin-01',
      dayNumber: 1,
      timeSlot: '09:30 AM',
      title: 'Arrival & Check-in at Le Marais Boutique Hotel',
      description: 'Drop luggage, freshen up and enjoy café au lait & croissants on the terrace.',
      location: 'Le Marais, Paris',
      cost: 0.0,
      category: 'Hotel',
      isCompleted: true,
    ),
    ItineraryItem(
      id: 'itin-02',
      dayNumber: 1,
      timeSlot: '01:00 PM',
      title: 'Louvre Masterpieces Guided Art Tour',
      description: 'Skip the line and explore the Italian Renaissance wing and Mona Lisa gallery.',
      location: 'Rue de Rivoli, Paris',
      cost: 65.0,
      category: 'Activity',
      isCompleted: true,
    ),
    ItineraryItem(
      id: 'itin-03',
      dayNumber: 1,
      timeSlot: '06:30 PM',
      title: 'Eiffel Tower Sunset Summit & Champagne',
      description: 'Sunset ascent to the summit followed by illuminated city nightscape photos.',
      location: 'Champ de Mars, Paris',
      cost: 85.0,
      category: 'Activity',
      isCompleted: false,
    ),
    ItineraryItem(
      id: 'itin-04',
      dayNumber: 2,
      timeSlot: '10:00 AM',
      title: 'Montmartre Artists Square & Sacré-Cœur Basilica',
      description: 'Walk through cobblestone streets, view portrait painters and panoramic hilltop view.',
      location: 'Montmartre, Paris',
      cost: 20.0,
      category: 'Sightseeing',
      isCompleted: false,
    ),
    ItineraryItem(
      id: 'itin-05',
      dayNumber: 2,
      timeSlot: '07:00 PM',
      title: 'Bateaux Parisiens Gourmet Dinner Cruise',
      description: '4-course French culinary dinner cruise drifting past Notre-Dame and Île de la Cité.',
      location: 'Port de la Bourdonnais, Paris',
      cost: 110.0,
      category: 'Dining',
      isCompleted: false,
    ),
    ItineraryItem(
      id: 'itin-06',
      dayNumber: 3,
      timeSlot: '08:30 AM',
      title: 'High-Speed TGV Train to Zurich HB',
      description: 'Scenic 4-hour rail journey through Eastern France and Swiss countryside.',
      location: 'Gare de Lyon -> Zurich HB',
      cost: 140.0,
      category: 'Transport',
      isCompleted: false,
    ),
  ];

  // Expenses for Trip 1
  static final List<Expense> dummyExpenses = [
    Expense(
      id: 'exp-01',
      tripId: 'trip-001',
      title: 'Roundtrip Flights & TGV Train Tickets',
      amount: 680.0,
      category: 'Flights',
      date: DateTime.now().subtract(const Duration(days: 10)),
      paymentMethod: 'Amex Platinum',
      notes: 'Air France + SBB Swiss Rail',
    ),
    Expense(
      id: 'exp-02',
      tripId: 'trip-001',
      title: 'Boutique Hotels & Alpine Chalets (8 Nights)',
      amount: 820.0,
      category: 'Stays',
      date: DateTime.now().subtract(const Duration(days: 6)),
      paymentMethod: 'Visa Signature',
      notes: 'Paris (3N), Zurich (2N), Rome (3N)',
    ),
    Expense(
      id: 'exp-03',
      tripId: 'trip-001',
      title: 'Michelin-guide Dining & Trattorias',
      amount: 350.0,
      category: 'Food',
      date: DateTime.now().subtract(const Duration(days: 2)),
      paymentMethod: 'Apple Pay',
      notes: 'Fine dining and local espresso bars',
    ),
    Expense(
      id: 'exp-04',
      tripId: 'trip-001',
      title: 'Museum Passes, Tours & Skip-the-Line',
      amount: 240.0,
      category: 'Activities',
      date: DateTime.now().subtract(const Duration(days: 1)),
      paymentMethod: 'Credit Card',
      notes: 'Louvre, Eiffel, Colosseum & Vatican',
    ),
  ];

  // Multi-City Stops for Trip 1
  static final List<TripStop> dummyStopsTrip1 = [
    TripStop(
      id: 'stop-01',
      city: dummyCities[0], // Paris
      arrivalDate: DateTime.now().add(const Duration(days: 14)),
      departureDate: DateTime.now().add(const Duration(days: 17)),
      budget: 850.0,
      hotelName: 'Le Marais Boutique Hotel',
      transportMode: 'Flight',
    ),
    TripStop(
      id: 'stop-02',
      city: dummyCities[1], // Zurich
      arrivalDate: DateTime.now().add(const Duration(days: 17)),
      departureDate: DateTime.now().add(const Duration(days: 19)),
      budget: 700.0,
      hotelName: 'Lake Zurich Grand Residence',
      transportMode: 'High Speed Train',
    ),
    TripStop(
      id: 'stop-03',
      city: dummyCities[2], // Rome
      arrivalDate: DateTime.now().add(const Duration(days: 19)),
      departureDate: DateTime.now().add(const Duration(days: 22)),
      budget: 800.0,
      hotelName: 'Piazza Navona Heritage Suites',
      transportMode: 'Flight',
    ),
  ];

  // Full Trip List
  static final List<Trip> dummyTrips = [
    Trip(
      id: 'trip-001',
      title: 'European Grand Tour 2026',
      description: 'Parisian cafes, Swiss alpine peaks, and Roman ancient monuments in one seamless multi-city adventure.',
      coverImageUrl:
          'https://images.unsplash.com/photo-1499856871958-5b9627545d1a?w=900&auto=format&fit=crop&q=80',
      startDate: DateTime.now().add(const Duration(days: 14)),
      endDate: DateTime.now().add(const Duration(days: 22)),
      budget: 3200.0,
      spent: 2090.0,
      currency: 'USD',
      status: TripStatus.upcoming,
      stops: dummyStopsTrip1,
      itineraryItems: dummyItinerary,
      expenses: dummyExpenses,
      isPublic: true,
      createdBy: 'Alex Morgan',
      travelersCount: 2,
    ),
    Trip(
      id: 'trip-002',
      title: 'Tropical Bali & Island Sanctuary',
      description: 'Immersive wellness retreat, waterfalls, sacred temples, and sunrise volcano trekking.',
      coverImageUrl:
          'https://images.unsplash.com/photo-1537996194471-e657df975ab4?w=900&auto=format&fit=crop&q=80',
      startDate: DateTime.now().add(const Duration(days: 45)),
      endDate: DateTime.now().add(const Duration(days: 52)),
      budget: 1800.0,
      spent: 650.0,
      currency: 'USD',
      status: TripStatus.upcoming,
      stops: [
        TripStop(
          id: 'stop-b1',
          city: dummyCities[4], // Bali
          arrivalDate: DateTime.now().add(const Duration(days: 45)),
          departureDate: DateTime.now().add(const Duration(days: 52)),
          budget: 1800.0,
          hotelName: 'Maya Ubud Resort & Spa',
          transportMode: 'Flight',
        ),
      ],
      itineraryItems: const [],
      expenses: const [],
      isPublic: true,
      createdBy: 'Alex Morgan',
      travelersCount: 1,
    ),
    Trip(
      id: 'trip-003',
      title: 'Tokyo Neon & Kyoto Heritage',
      description: 'Futuristic innovation meets ancient bamboo groves and traditional ryokans.',
      coverImageUrl:
          'https://images.unsplash.com/photo-1503899036084-c55cdd92da26?w=900&auto=format&fit=crop&q=80',
      startDate: DateTime.now().subtract(const Duration(days: 90)),
      endDate: DateTime.now().subtract(const Duration(days: 80)),
      budget: 3500.0,
      spent: 3420.0,
      currency: 'USD',
      status: TripStatus.completed,
      stops: [
        TripStop(
          id: 'stop-t1',
          city: dummyCities[3], // Tokyo
          arrivalDate: DateTime.now().subtract(const Duration(days: 90)),
          departureDate: DateTime.now().subtract(const Duration(days: 80)),
          budget: 3500.0,
          hotelName: 'Shinjuku Prince Hotel',
          transportMode: 'Flight',
        ),
      ],
      itineraryItems: const [],
      expenses: const [],
      isPublic: true,
      createdBy: 'Alex Morgan',
      travelersCount: 2,
    ),
  ];
}
